src/ameba/ast/visitors/implicit_return_visitor.cr

```diff
@@ -324,20 +304,11 @@ module Ameba::AST
 
       # Regex literals either contain string literals or string interpolations,
       # both of which are "captured" by the parent regex literal
-      incr_stack { node.value.accept(self) }
+      incr_stack { node.accept_children(self) }
 
       false
     end
 
-    def visit(
-      node : Crystal::BoolLiteral | Crystal::CharLiteral | Crystal::NumberLiteral |
-             Crystal::StringLiteral | Crystal::SymbolLiteral | Crystal::ProcLiteral,
-    ) : Bool
-      report_implicit_return(node)
-
-      true
-    end
-
```

Comment:
Hard to follow why this was removed as well.

Reply:
It was replaced by definition with an union instead of duplicating it for each argument type.

Reply:
They're covered by the untyped visit method now https://github.com/crystal-ameba/ameba/pull/599/files#diff-8eaca1fca4746288c3e629e6af8c7e569d6b72cfad5cf423566e14949bbef348R368

Reply:
Ok, thanks